.gitignore

```diff
@@ -25,3 +25,5 @@ yarn-error.log*
 __pycache__
 
 .vscode
+
+src/data
```

Comment:
do we not want to share this data? or is this because the files are going to be massive

Reply:
Files are going to get massive